packages/taiko-client/prover/anchor_tx_validator/anchor_tx_validator.go

```diff
@@ -45,13 +47,15 @@ func (v *AnchorTxValidator) ValidateAnchorTx(tx *types.Transaction) error {
 		return fmt.Errorf("invalid TaikoL2.anchor transaction sender: %s", sender)
 	}
 
-	method, err := encoding.TaikoL2ABI.MethodById(tx.Data())
-	if err != nil {
-		return fmt.Errorf("failed to get TaikoL2.anchor transaction method: %w", err)
+	var method *abi.Method
+	if method, err = encoding.TaikoAnchorABI.MethodById(tx.Data()); err != nil {
+		if method, err = encoding.TaikoL2ABI.MethodById(tx.Data()); err != nil {
+			return fmt.Errorf("failed to get TaikoL2.anchor transaction method: %w", err)
+		}
 	}
-	if method.Name != "anchor" && method.Name != "anchorV2" {
+	if method.Name != "anchor" && method.Name != "anchorV2" && method.Name != "anchorV3" {
 		return fmt.Errorf(
-			"invalid TaikoL2.anchor transaction selector, expect: %s, actual: %s",
+			"invalid TaikoL2.anchorV2 / TaikoAnchor.anchorV3 transaction selector, expect: %s, actual: %s",
 			"anchor / anchorV2",
```

Comment:
should be `anchor / anchorV2 / anchorV3`

Reply:
Changed: https://github.com/taikoxyz/taiko-mono/pull/18746/commits/e0912fdf83207815d646c59a0e3a01607674d9e4